framework/src/media/MediaRecorderImpl.cpp

```diff
@@ -60,7 +63,8 @@ void MediaRecorderImpl::createRecorder(recorder_result_t& ret)
 		return notifySync();
 	}
 
-	audio_manager_result_t result = init_audio_stream_in();
+	ASSERT(gInCardId >= 0);
```

Comment:
I think it checks twice.

Reply:
Each audio manager function must check the wrong card ID.

Reply:
I see.
The "ASSERT" is removed.